src/lua/crypto_ethereum.lua

```diff
@@ -348,7 +373,7 @@ end
 
 local encode_tuple
 
-local function encode(t, arg)
+function encode(t, arg)
```

Comment:
This should be kept local, we export in with `ETH.abi_encode = encode`

Reply:
At line 230, the function encode is indeed initialised as "local", and then it is declared and defined at 376.